iDiff/tests/fileDiff_test_processor.py

```diff
@@ -27,7 +27,7 @@ def _trim_file_names(files):
 
 
 def _trim_layer_hash(filename):
-    hash_match = re.match(r'^([a-z|0-9]{64})/', filename)
+    hash_match = re.match(r'^([a-z|0-9])/', filename)
```

Comment:
When I was still saving the docker registry pulled info to tars it was no longer a 64 character preface to the first "/".